### ISSUE TITLE
strikethrough doesn't work for GFM parser

### ISSUE THREAD

**Issue by CGarces** [gettalong/kramdown#304]
GitHub Flavored Markdown supports strikethrough but the GFM parser included with kramdown doesn't support it.

In a [sample site](https://github.com/CGarces/strikethrough_issue) with the default github values

```
 kramdown:
   input: GFM
   hard_wrap: false
```

The strikethrough not work [see rendered page](http://cgarces.github.io/strikethrough_issue/)

Now is posible to made the strikethrough works using redcarpet.

```
markdown: redcarpet
redcarpet:
  extensions: ["strikethrough"]
```
But it'll be not possible in the future due the github pages migration to kramdown.

This is related with the PR #184

**Comment by mzlogin** [comment]
+1

**Comment by shuhrat** [comment]
+1

**Comment by atruskie** [comment]
I'm experience this problem as well. Any help appreciated.

**Comment by parkr** [comment]
Hey @gettalong! Just wanted to see if you have thoughts here. We're very interested in working with you to bring kramdown up to speed with the current formulation of GFM. If you can point me in the right direction, I might be able to take a crack at it. Thanks!

**Comment by benbalter** [comment]
> f you can point me in the right direction, I might be able to take a crack at it.

@parkr https://github.com/gettalong/kramdown/pull/184 may point you down the right path, if you haven't :eyes:'d it already.

**Comment by gettalong** [comment]
Fixed with commit ac71ef7, see #307 for more info.
